doc/scss/index.scss

```diff
@@ -0,0 +1,172 @@
+/**
+ * Styling sheet for the scikit-learn landing page. This should be loaded only for the
+ * landing page.
+ */
```

Comment:
It would be helpful to always add the following at the top of our .scss files to make it easier to on-board contributors not familiar with CSS preprocessors and their integration with sphinx.

```suggestion
 *
 * This file is compiled into styles/index.css by sphinxcontrib.sass.
 * https://sass-lang.com/guide/
 */
```

Reply:
@Charlie-XIAO do you agree with this? If so we can commit and merge the PR.